test-build.ps1

```diff
@@ -0,0 +1,32 @@
+Param( [string]$node_version, [string]$variant )
+
+$ErrorActionPreference = 'Stop'
+
+Function Get-Version {
+  Get-ChildItem -directory | Where-Object { ($_.Name -ne "docs") -and ($_.Name -ne "keys") -and ($_.Name -notcontains "chakracore") }
+}
+
+Function Build {
+  param ( $version, $variant, $tag )
+
+  $full_tag="$tag-$variant"
+
+  Write-Output "Building node:$full_tag"
+  docker build -t node:$full_tag $version/windows/$variant
+  if ( $? -eq $FALSE ) {
+    Write-Error "Build of $full_tag failed!"
+  }
+  Write-Output "Testing of $full_tag"
+  $path = (Get-Location).path.replace('\', '/')
+  docker run --rm -v "$path/:C:/Users/ContainerAdministrator/docker-node/" node:"$full_tag" powershell "C:\Users\ContainerAdministrator\docker-node\test-image.ps1" "$tag"
```

Comment:
Beginning with nanoserver 1709 the container default user no longer is ContainerAdministrator.
Mounting to eg. /docker-node

```powershell
docker run --rm -v "$path/:C:/docker-node/" node:"$full_tag" powershell "C:\docker-node\test-image.ps1" "$tag"
```

should fix this.